src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsRulesSpec.groovy

```diff
@@ -988,4 +1000,139 @@ class PriceFloorsRulesSpec extends PriceFloorsBaseSpec {
         where:
         enforcePbs << [true, null]
     }
+
+    def "PBS should use correct rule for each bidder when bidder is define with valid bid floor value"() {
```

Comment:
"PBS should use correct rule for each bidder when bidder is define with valid bid floor value" -> "PBS should use correct rule for each bidder when a bidder is defined with valid bid floor value"